cmd/tbcd/tbcd.go

```diff
@@ -93,6 +93,12 @@ var (
 			Help:         "address and port tbcd pprof listens on (open <address>/debug/pprof to see available profiles)",
 			Print:        config.PrintAll,
 		},
+		"TBC_SEEDS": config.Config{
+			Value:        &cfg.Seeds,
+			DefaultValue: []string{},
+			Help:         "list of tbc seeds, format of <host>:<port>",
```

Comment:
I think we could make this help message more detailed by saying these are used for Bitcoin P2P, and specifying that localnet requires a single host:port instead of a list of seeds.
```suggestion
			Help:         "list of seed domains for Bitcoin P2P, in the format '<host>:<port>' (for localnet, must be a single host:port)",
```